BankSDK/GiniBankSDKExample/GiniBankSDKExample/SettingsViewController/SettingsViewController.swift

```diff
@@ -64,487 +59,105 @@ final class SettingsViewController: UIViewController {
         view.addGestureRecognizer(tapGesture)
     }
 
-	private func configureNavigationBar() {
-		navigationBar.isTranslucent = false
-		navigationBar.tintColor = ColorPalette.raspberryPunch
-		let closeButton = UIBarButtonItem(title: "Close",
-										  style: .plain,
-										  target: nil,
-										  action: #selector(didSelectCloseButton))
-		closeButton.target = self
-		navigationBarItem.leftBarButtonItem = closeButton
-	}
+    private func configureNavigationBar() {
+        navigationBar.isTranslucent = false
+        navigationBar.tintColor = ColorPalette.raspberryPunch
+        let closeButton = UIBarButtonItem(title: "Close",
+                                          style: .plain,
+                                          target: nil,
+                                          action: #selector(didSelectCloseButton))
+        closeButton.target = self
+        navigationBarItem.leftBarButtonItem = closeButton
+    }
+
+    private func configureTableView() {
+        tableView.dataSource = self
+        tableView.delegate = self
+
+        tableView.separatorStyle = .none
+        tableView.allowsSelection = false
+        tableView.showsVerticalScrollIndicator = false
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.estimatedRowHeight = 65
+
+        if #available(iOS 15.0, *) {
+            tableView.sectionHeaderTopPadding = 0
+        }
 
-	private func configureTableView() {
-		tableView.dataSource = self
-		
-		tableView.separatorStyle = .none
-		tableView.allowsSelection = false
-		tableView.showsVerticalScrollIndicator = false
-		tableView.rowHeight = UITableView.automaticDimension
-		tableView.estimatedRowHeight = 65
-		
-		if #available(iOS 15.0, *) {
-			tableView.sectionHeaderTopPadding = 0
-		}
-		
-		tableView.register(SwitchOptionTableViewCell.self)
-		tableView.register(SegmentedOptionTableViewCell.self)
-		tableView.register(InfoTableViewCell.self)
+        tableView.register(SwitchOptionTableViewCell.self)
+        tableView.register(SegmentedOptionTableViewCell.self)
+        tableView.register(InfoTableViewCell.self)
         tableView.register(CredentialsTableViewCell.self)
         tableView.register(UpdateUserDefaultsCell.self)
+        tableView.register(SettingsHeaderView.self, forHeaderFooterViewReuseIdentifier: SettingsHeaderView.identifier)
```

Comment:
[request] In the example app we have some util methods to get rid of having hardcoded reuse identifiers for every view.
Let's discuss this.